skycoin/skywire
# Update to Angular 13

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- Angular was updated to v13.
- TSLint was replaced with ESLint, as TSLint was removed in Angular 13. Several changes were made to solve problems returned by the new linter.

How to test this PR:
Run `make install-deps-ui`. It should be possible to compile the UI again after that and it should work normally.
